shoov/modules/custom/shoov_screenshot/shoov_screenshot.module

```diff
@@ -22,6 +22,13 @@ function shoov_screenshot_menu() {
     'type' => MENU_CALLBACK,
   );
 
+  $items['shoov/images/styles/%image_style'] = array(
```

Comment:
@amitaibu This was missing. Using callback from image.module, that will create (of take existing) style images.